python/cudf/cudf/tests/test_indexing.py

```diff
@@ -2392,3 +2392,22 @@ def test_loc_iloc_setitem_col_slice_non_cupy_types(indexer, dtype):
     getattr(df_pd, indexer)[:, 0] = getattr(df_pd, indexer)[:, 0]
     getattr(df_cudf, indexer)[:, 0] = getattr(df_cudf, indexer)[:, 0]
     assert_eq(df_pd, df_cudf)
+
+
+@pytest.mark.parametrize("indexer", ["iloc", "loc"])
+@pytest.mark.parametrize(
+    "column_slice",
+    [
+        slice(None),
+        slice(0, 0),
+        slice(0, 1),
+        slice(1, 0),
+        slice(0, 2, 2),
+    ],
+)
+def test_slice_empty_columns(indexer, column_slice):
+    df_pd = pd.DataFrame(index=[0, 1, 2])
+    df_cudf = pd.DataFrame(index=[0, 1, 2])
```

Comment:
```suggestion
    df_cudf = cudf.DataFrame(index=[0, 1, 2])
```

I'm assuming 😄

Reply:
Yikes... I'll blame this on my dev env being a bit messed up right now.